### PR TITLE
Fix missing spaces with help for `-m compileall -o`, revisit

### PR DESCRIPTION
**Pull request by blueyed**

### CODE REVIEW

File: Lib/compileall.py [hunk 1]
```diff
@@ -367,9 +367,9 @@ def main():
                               'environment variable is set, and '
                               '"timestamp" otherwise.'))
     parser.add_argument('-o', action='append', type=int, dest='opt_levels',
```
Comment (ambv) [inline Lib/compileall.py:369]: Maybe fix the `dest=` here too to be singular.

**Comment by ambv** [comment]
An issue and NEWS entry would be nice to have as well.

File: Lib/compileall.py (hunk 1 above)
Comment (blueyed) [reply Lib/compileall.py:369]: It is actually a list.
I've fixed the fixup.

**Review by blueyed — COMMENTED** [review]
> An issue and NEWS entry would be nice to have as well.

I do not think it is worth it, but can add it if required really, of course.

**Review by serhiy-storchaka — APPROVED** [review]

**Pull request merged**